examples/room-manager/src/plugins/fishjam.ts

```diff
@@ -31,7 +42,7 @@ export class RoomService {
 
     if (!peer) {
       fastify.log.info({ name: 'Creating peer' });
-      return await this.createPeer(roomName, username);
+      return await createPeer(roomName, username);
```

Comment:
**nitpick:** `await` is redundant in this context.

Reply:
out of scope, but true, fixed